### DISCUSSION TITLE
SchedulesDirect upcoming DirecTV channel changes, removing analog redundancy.

### DISCUSSION THREAD

**Discussion by eelpout** [OpenSageTV/Community#99] [General]
FYI. received this today from SchedulesDirect. Maybe someone already posted this, if so sorry for the duplicate.

Our data provider Gracenote will be updating a portion of the US lineups to 
reflect the market. Analog lineups for Satellite, Cable and others will be
updated to digital and duplicate analog lineups will be removed.  

You should change your lineups as reflected below before **December 11th** as 
Gracenote will be starting to make changes then.

old lineup                    
USA-DITV807-DEFAULT       

new lineup
USA-DITV807-X

If you have any questions, please use the "lineup support" link on our website.

Sincerely, Schedules Direct

Schedules Direct
https://www.schedulesdirect.org/

**Comment by RJ-Make** [comment]
I don't see those referenced in my setup, what does this mean? (here's my setup)

![2023-12-04_7-07-10](https://github.com/OpenSageTV/Community/assets/8496835/17170c53-3b40-44b1-8b89-1bd23e6125b2)

![2023-12-04_7-08-29](https://github.com/OpenSageTV/Community/assets/8496835/e1ef2237-73fb-4422-9ce4-64c11d157889)

**Comment by evilattorney** [comment]
I received a similar email too, but am confused about what to do next.  I tried looking for a second Directv lineup in my area, but don't see it.   Has anyone figured out what needs to be done?

**Comment by eelpout** [comment]
You'll need to go to your online account at schedulesdirect and see what is selected. Mine is currently "USA-DITV807-" I'm going to check with them, but I assume that's the same as "-X"

**Reply by evilattorney** [reply]
Are you sure? I thought the lineups in Sage use the JSON listing and not the old version that you select the lineup on the SD account page.  But I don't see a different option when selecting in Sage, only the Directv listing I am currently using, so it's confusing.  I may send a message to their support to figure this out.

**Comment by eelpout** [comment]
had to refresh my brain as this has been years since I did this. 

[here is the Wiki page](https://github.com/google/sagetv/wiki/Schedules-Direct-EPG) for setting up SD with SageTV. It does look like one will have to reset within SageTV itself, so you are correct. and for reference [here is the old forums page](https://forums.sage.tv/forums/showthread_t_63532.html?t=63532) on SD.

I haven't changed mine yet, kind of afraid to touch it without first backing everything up. ;)

**Reply by evilattorney** [reply]
I tried doing this but there is still only one option in Sage for me to select (Directv in Los Angeles) so it isn't clear if I am just again selecting the old lineup or it is the new lineup.  In other words, it doesn't provide the more detailed name that SD uses and that was in the email they sent.

**Reply by eelpout** [reply]
I'd check with SD then; their support is pretty good. 

But this likely means you'll be fine. AFAIK, all this change does is consolidate all the duplicate analog channels which are on the same number as the HD one. So, if one already has only the HD versions selected currently in the lineup in Sage, one should be OK.

**Comment by LoisGNS** [comment]
I didn't get such an email. Could it just be users of DirectTV who are affected? I use OTA and Comcast/Xfinity.

**Reply by eelpout** [reply]
yes, it's about an analog channel consolidation on DirecTV only.

**Comment by eelpout** [comment]
argh. lost what I typed in before.

I'm only seeing a single entry for my zip code for DirecTV on the SD website now, even though the email says there is more than one. So I think one would be fine if seeing the same thing. Most, if not all, channels in my area for analog and HD have the same number now anyway.

We really won't know until the 11TH when Gracenote starts to make the channel changes what is going to happen. One can start to check your local lineup at Zap2It.com and when the analog duplication disappears (attached a sample of the duplication below), then I'd probably force a rescan on your channel lineup on the source in SageTV to make sure everything is in sync.

![directv](https://github.com/OpenSageTV/Community/assets/17970786/7dcb39fe-09aa-45eb-b108-eb97b0297611)

**Comment by eelpout** [comment]
I know it's the first day that Gracenote will start their changes, but so far mine is the same.

**Comment by Polypro1** [comment]
Just went to the old forum - RIP.  I got the SD email too, just tried to figure out the new DITV505-X....no joy. If it's actually needed, and someone figures out what to do - post it up please. Thanks!

**Comment by evilattorney** [comment]
I still have two weeks of EPG data, so I assume that its all being taken care of automatically, but will keep monitoring it.

**Comment by eelpout** [comment]
is anyone else seeing any changes in the guide at Zap2It.com?I think the change is happening now, as I see no DirecTV listings at all now for my zip code.

![Screenshot 2023-12-15 110425 (Small)](https://github.com/OpenSageTV/Community/assets/17970786/97163185-4920-4c65-943c-559b338a78d4)

**Reply by eelpout** [reply]
can confirm on SchedulesDirect that one of my lineups is now dead: DITV807:-. 

the active one for my zip is: DITV807-X

I've sent a problem report to SD, asking them to remove the old defunct lineup entries so we can reselect to make sure we get the correct JSON lineups. 

if you look in the SageTV.properties file, what I believe are the entries for the lineups are here:

```
epg_data_sources/51839341/epg_name=DIRECTV San Francisco - Satellite - San Francisco (sdepg)
epg_data_sources/51839341/epg_sd_name=USA-DITV807-DEFAULT
```

I'm tempted to shutdown and edit this entry to point to what SD has recommended 'DITV807-X' and see what happens.

**Reply by eelpout** [reply]
**EDIT**:  heard back from SD, make sure to use the "USA" prefix, it denotes the JSON service. so in my case it is "USA-DITV807-X" the other is for XML.

I would recommend going to your SD account page and see the listings for your zip code and look for the new lineup ID that works by clicking on the preview link. 

Then you can edit (with service stopped) the `epg_data_sources` entry for your DirecTV source.

**Reply by eelpout** [reply]
What I ended up doing was going into SageTV setup for the sources for my two DirecTV boxes, removing the existing lineup and then adding a new one that contains "USA-DITV807-X" in its name. seems to be working OK so far.

**Comment by evilattorney** [comment]
Ok, starting today when I went to choose a new EPG lineup in Sage it listed "USA-DITV803-X" instead of the generic DirecTV name for my city.  That was the name SD told me to use in the email.  I guess it just took time for that EPG name to get in their system?  Strangely, I am seeing the duplicate analog and digital entries for some channels, which I thought this lineup change was attempting to address by removing the analog entries.  Oh well.

**Reply by eelpout** [reply]
I'm not sure when these new JSON entries were first added by SD, they appeared a couple weeks ago and were populated with listings. I changed mine over yesterday. 

I'm with you on the duplications though, I thought that was the whole point of the exercise. Unless maybe I misunderstood what they meant by _duplication_. ???  Since I'm having a small dialogue with SD, I asked them to clarify this. 

The Zap2It listings seem unchanged as well. Could be Gracenote is just dragging their feet doing the listings update on their end. SD said they are having some issues with Gracenote data themselves and are issuing support tickets to them. Shrug.

**Comment by Polypro1** [comment]
Thanks for hashing this out - I still have data probs won't mess with it until I see a problem.
